src/backend/gsroa.c

```diff
@@ -198,6 +270,13 @@ void sliceStructs()
             sia2[si + n].canSlice = false;
             if (sia[si].canSlice)
             {
+                if (!sia[si].accessSlice)
+                {
+                    // If never did access it as a slice, don't slice
+                    sia[si].canSlice = false;
```

Comment:
Well, but here it could be sliced but shouldn't be done (b/c it's not used). Anyhow...